src/openklant/components/klanten/api/serializers.py

```diff
@@ -315,6 +316,11 @@ def __init__(self, *args, **kwargs):
                 self.fields[custom_field].help_text
             )
 
+    def validate_klantnummer(self, value):
```

Comment:
It looks like it duplicates klantnummer generation in the `Klant.save` method

Reply:
Nice catch, I couldn't get it to work on the serializer so moved this to the save(). Tests should pass without this, so I've removed it.